restli-tools/src/test/java/com/linkedin/restli/tools/clientgen/TestRestRequestBuilderGeneratorEntryPoint.java

```diff
@@ -0,0 +1,141 @@
+package com.linkedin.restli.tools.clientgen;
+
+import com.linkedin.data.schema.generator.AbstractGenerator;
+import com.linkedin.pegasus.generator.PegasusDataTemplateGenerator;
+import com.linkedin.restli.internal.common.RestliVersion;
+import com.linkedin.restli.tools.ExporterTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+
+public class TestRestRequestBuilderGeneratorEntryPoint {
```

Comment:
Is this basically just a copy of the existing test, but using PSVM instead? Would it be possible to keep this in the existing test class, or would that be too messy you think?

Reply:
Due to the messiness of the sysprop backup/restore, I chose to extract it to a separate class. I'm still worried about parallelism or unwanted side-effects of this, so we should kill this.